test-network/network.sh

```diff
@@ -283,10 +291,13 @@ function networkDown() {
   # Don't remove the generated artifacts -- note, the ledgers are always removed
   if [ "$MODE" != "restart" ]; then
     # Bring down the network, deleting the volumes
+    docker volume prune -f
```

Comment:
I don't think we want this as it will prune volumes outside of fabric test-network which a user may still want. It should be done by docker-compose

Reply:
Well `docker-compose` *should* do it, it fails fast. So it can fail for example if something goes wrong.. eg the network can't be deleted, or if you've restarted the machine, the containers aren't running etc..

Though adjusting this to a specific delete of the required volumes would be an idea

Reply:
@davidkel I've replaced with specific `docker volume rm` commands